internal/parseutil/filereader_test.go

```diff
@@ -0,0 +1,48 @@
+package parseutil
```

Comment:
This is a straight-forward "no dependencies" package, so let's put this in package `parseutil_test` so we're not testing internals, and only test the exported interface.